shop/admin.py

```diff
@@ -1,3 +1,75 @@
 from django.contrib import admin
+from .models import *
 
-# Register your models here.
+
+class Customer_Admin(admin.ModelAdmin):
+    list_display = ('user', 'phone_number', 'address')
+    list_display_links = ('user',)
+    search_fields = ('user', 'phone_number', 'address')
+
+
+admin.site.register(Customer, Customer_Admin)
+
+
+class Seller_Admin(admin.ModelAdmin):
+    list_display = ('user', 'seller_name', 'address')
+    list_display_links = ('user',)
+    search_fields = ('user', 'seller_name', 'address')
+
+
+admin.site.register(Seller, Seller_Admin)
+
+
+class Category_Admin(admin.ModelAdmin):
```

Comment:
I think you can use `@admin.register(Category)`
`from django.contrib import admin`